src/test/java/com/google/api/codegen/testdata/discoveries/go/go_storage.v1.json.baseline

```diff
@@ -1474,6 +1474,8 @@ func main() {
     // TODO: Add desired fields of the request body.
   }
 
+  // TODO: Use `Download` in place of `Do` below to download media content. For more information, see:
+  // https://godoc.org/google.golang.org/api/storage/v1
```

Comment:
Is there a more specific media download URL we could use, or is this the best we can do for Go? The other languages where you specify URLs appear to have it.

Maybe a link to the method, eg `https://godoc.org/google.golang.org/api/storage/v1#ObjectsComposeCall.Download`?